src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/InputEmptyLineSeparatorFormerException.java

```diff
@@ -1,6 +0,0 @@
-package com.puppycrawl.tools.checkstyle.checks.whitespace;
-
-public class InputEmptyLineSeparatorFormerException
-{
-
-}
```

Comment:
In order to move the files wont the first command suffice since git mv oldname newname will rename the files? Almost all the input files are named InputEmptyLineSeparator[*].java since the test name is EmptyLineSeparatorCheckTest.java as per the parent Issue #3891 except InputPrePreviousLineEmptiness.java and files in package-info/*